src/contentAPPXbundle.ts

```diff
@@ -30,7 +30,7 @@ export class AppxBundleContent extends ContentBase {
 
         this.CheckManifestForAppx(manifestData, fileList);
         if(this.appxName) {
-            return await this.readAppx(fileList, tempDir);
```

Comment:
If I remember correctly, if the appx is found in the manifest then we can extract all the data from it, so there is no need to continue the flow.

Reply:
If return here, properties such as build version will not be assigned which makes the unit test failed.

Reply:
Ok removing the return means:
1. All the languages are taken from the appxbundle and not from the  inner appx. 
2. If icon wasn't extracted from the inner appex, we will try to extract it from the bundle. 

Both options makes sense to me. So let's keep it the way it is.